web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js

```diff
@@ -46,7 +46,9 @@ context('Create and delete consumer with limit-count plugin form', () => {
     consumerName: 'test_consumer',
     description: 'desc_by_autotest',
     createConsumerSuccess: 'Create Consumer Successfully',
-    deleteConsumerSuccess: 'Delete Consumer Successfully'
+    deleteConsumerSuccess: 'Delete Consumer Successfully',
+    redisClusterName: 'Please Enter redis_cluster_name',
```

Comment:
```suggestion
    redisClusterName: 'Please Enter name of Redis Cluster',
```

Reply:
i can understand your suggestion, and thanks for your review. 
Only the test contents need consistent with the required tips contents, and the style of all the tips contents for the plugin is like this(redis_cluster_name).